src/app.rs

```diff
@@ -74,6 +74,10 @@ pub enum ClaimsScope {
     Generate,
     // Permission to list builds and to download a build repo.
     Download,
+    // Permission to republish an app (take it from the repo, re-run the publish hook, and publish it back). Should not
+    // be given to untrusted parties.
```

Comment:
I think we need some explanation of what "trusted" means here to guide use.  Maybe that belongs in the code comment, maybe in a README?

Reply:
Added to the token section in README.md:

> Some token privileges are for managing flat-manager and shouldn't be
given to third parties who are just uploading apps. The token privileges
are described in the ClaimsScope enum in `app.rs`.